clang/test/Sema/integral-to-ptr.c

```diff
@@ -0,0 +1,3 @@
+// RUN: %clang_cc1 %s -verify -fsyntax-only -std=c11
+
+int x(void) { e: b: ; return &&e - &&b < x; } // expected-warning {{ordered comparison between pointer and integer ('long' and 'int (*)(void)')}}
```

Comment:
```suggestion
int x(void) { e: b: ; return &&e - &&b < x; } // expected-warning {{ordered comparison between pointer and integer}}
```
On Windows, this is a `long long`, not a `long`, which is why the test is currently failing.